src/Encamina.Enmarcha.SemanticKernel/MemoryStoreExtender.cs

```diff
@@ -0,0 +1,165 @@
+﻿// Ignore Spelling: Upsert
+
+using System.Collections.ObjectModel;
+using System.Runtime.CompilerServices;
+using System.Text.Json;
+
+using Encamina.Enmarcha.SemanticKernel.Abstractions;
+using Encamina.Enmarcha.SemanticKernel.Abstractions.Events;
+
+using Microsoft.Extensions.Logging;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Embeddings;
+using Microsoft.SemanticKernel.Memory;
+
+namespace Encamina.Enmarcha.SemanticKernel;
+
+/// <summary>
+/// Extender that provides some CRUD operations over memories with multiple chunks that need to be extended by an <see cref="IMemoryStore"/>, using batch operations.
```

Comment:
It would be better as:
"_This class provides some additional CRUD operations over memories with multiple chunks that extends operations from a `<see cref="IMemoryStore"/>`_"

Reply:
Thank you for your contribution. I will update it.